### PR TITLE
slot mapping examples in docs adaptions

### PR DESCRIPTION
**Pull request by melindaloubser1**
**Proposed changes**:
- Update examples & surrounding explanation in docs pages that reference slot mappings. 
Closes https://github.com/RasaHQ/rasa/issues/10063 which was already partially addressed by https://github.com/RasaHQ/rasa/pull/10011

**Status (please check what you already did)**:
- [x] updated the documentation

### CODE REVIEW

**Review by rctatman — CHANGES_REQUESTED** [review]
typo in line 123 (conditons -> conditions)

File: docs/docs/business-logic.mdx [hunk 1]
```diff
@@ -110,42 +111,17 @@ pipeline:
...
+`deny`, it'll be `false`. 
 
-Since the form relies on certain [slots](domain.mdx#slots) being available, you need
-to add these slots to the domain. Slots filled by forms should usually not influence
-the conversation. Set `influence_conversation` to `false` to ignore their values during
-the conversation:
+However, the slot should only be set to `true` or `false` if the user was responding to the question, `Do you want to sit outside?`.
+To enforce this condition, the `conditions` for the `outdoor_seating` slot requires that `restaurant_form` is active and that the requested slot is `outdoor_seating`. 
+If there were no conditions and the user had sent a message with the `affirm` or `deny` intent earlier in the conversation,
+the `outdoor_seating` slot would already be filled when the form was activated. Therefore the form would not prompt the user for their outdoor seating preference. See [mapping conditons](./domain.mdx#mapping-conditions) for more information.
```
Comment (melindaloubser1) [inline docs/docs/business-logic.mdx:123]:
```suggestion
the `outdoor_seating` slot would already be filled when the form was activated. Therefore the form would not prompt the user for their outdoor seating preference. See [mapping conditions](./domain.mdx#mapping-conditions) for more information.
```

File: docs/docs/business-logic.mdx [hunk 2]
```diff
@@ -74,32 +85,22 @@ slots:
       entity: number
   outdoor_seating:
     type: bool
+    conditions:
+    - active_loop:
+      - restaurant_form
+    - requested_slot:
+      - outdoor_seating
```
Comment (melindaloubser1) [inline docs/docs/business-logic.mdx:92]:
```suggestion
    - active_loop: restaurant_form
       requested_slot: outdoor_seating
```

File: docs/docs/business-logic.mdx [hunk 3]
```diff
@@ -53,14 +53,25 @@ from the user:
...
+```yaml-rasa title="domain.yml"
+forms:
+  restaurant_form:
+    required_slots:
+        - cuisine
+        - num_people
+        - outdoor_seating
+```
+
+These slots need to be added to your domain's `slots` section, along with [slot mappings](forms.mdx#slot-mappings) which define how the slots can be filled. For any slot filled `from_entity`, the entity also needs to be added
```
[the remark this answers is not in the document]
Comment (melindaloubser1) [reply docs/docs/business-logic.mdx:67]: Yes, it should be updated when the other PR is merged

File: docs/docs/business-logic.mdx [hunk 4]
```diff
@@ -74,32 +85,20 @@ slots:
...
+    conditions:
+    - active_loop: restaurant_form
+      requested_slot: outdoor_seating
     mappings:
     - type: from_intent
       intent: affirm
       value: true
     - type: from_intent
       intent: deny
       value: false
```
Comment (melindaloubser1) [inline docs/docs/business-logic.mdx:97]:
```suggestion
    mappings:
    - type: from_intent
      intent: affirm
      value: true
      conditions:
       - active_loop: restaurant_form
          requested_slot: outdoor_seating
    - type: from_intent
      intent: deny
      value: false
      conditions:
      - active_loop: restaurant_form
         requested_slot: outdoor_seating
```

File: docs/docs/business-logic.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (melindaloubser1) [reply docs/docs/business-logic.mdx:67]:
```suggestion
These slots need to be added to your domain's `slots` section, along with [slot mappings](domain.mdx#slot-mappings) which define how the slots can be filled. For any slot filled `from_entity`, the entity also needs to be added
```

File: docs/docs/business-logic.mdx [hunk 5]
```diff
@@ -78,28 +89,19 @@ slots:
     - type: from_intent
       intent: affirm
       value: true
+      conditions:
+       - active_loop: restaurant_form
+          requested_slot: outdoor_seating
```
Comment (ancalita) [inline docs/docs/business-logic.mdx:94]: Nit-picking (could also be my poor eyesight 😂 ): there seems to be an extra whitespace before `requested_slot`, should this be aligned with `active_loop` to make it look pretty? 😄  same comment for L 100.

**Review by ancalita — APPROVED** [review]
🎉

File: docs/docs/business-logic.mdx (hunk 5 above)
Comment (melindaloubser1) [reply docs/docs/business-logic.mdx:94]: you're totally right will fix =)

Comment (melindaloubser1) [reply docs/docs/business-logic.mdx:94]:
```suggestion
         requested_slot: outdoor_seating
```

File: docs/docs/business-logic.mdx [hunk 6]
```diff
@@ -78,28 +89,19 @@ slots:
...
       value: false
-forms:
-  restaurant_form:
-    required_slots:
-        - cuisine
-        - num_people
-        - outdoor_seating
+      conditions:
+      - active_loop: restaurant_form
+         requested_slot: outdoor_seating
```
Comment (melindaloubser1) [inline docs/docs/business-logic.mdx:100]:
```suggestion
        requested_slot: outdoor_seating
```

**Pull request merged**